sparkpost.md

```diff
@@ -1,31 +1,24 @@
 # SparkPost
 
-:warning:
-**Never ever set up a *\*.renuoapp.ch* as a sending domain to a new created account. This will block the account on SparkPost.
-If you need this, choose the second option.** :warning:
-
-There are two options to set up SparkPost:
+Each app is using a separate subaccount under the main SparkPost account sparkpost+renuoapp@renuo.ch.
 
-* With a domain for the project and a separate SparkPost account (preferred, if possible)
-* Without an exisiting domain ==> via renuoapp.ch SparkPost account
+There are two paths when setting up the project on SparkPost:
 
-**Note:** With the second option, you are only able to send mails with **\*@renuoapp.ch** as your mail sender.
+* The domain is known: it should be set up and verified under the subaccount's sending domains.
+* The domain is not known: the emails will be sent with **\*@renuoapp.ch** as your mail sender
 
-## First Option
+:warning:
+Never use the primary account for the project, so that the whole account doesn't get suspended / blocked in case of compliance issues!
 
-There should only be one SparkPost account per project. This means, that *master*, *develop* and *testing* use the same one.
+There should only be one SparkPost subaccount per project. This means, that *master*, *develop* and *testing* use the same one.
```

Comment:
This means, that we could break _master_ mail-sending by sending "abusive" emails from _develop_ and _testing_, right?

What about having one subaccount for all develop and one subaccount for all testing deployments? Something like sparkpost+develop@renuo.ch and sparkpost+testing@renuo.ch?

Reply:
I wrote somthing similar just now :) https://github.com/renuo/applications-setup-guide/pull/254#discussion_r569210519

Reply:
I'd propose to use mailtrap for testing and develop, but this is not always possible: it also depends on the customer.

Reply:
That's also fine for me, I can add it as an alternative. I think it's not necessary to split develop and testing sparkpost accounts, so I'd stick to either sparkpost+master@renuo.ch and sparkpost+develop@renuo.ch (which covers also testing), or keep sparkpost+renuoapp@renuo.ch as it is for develop / testing and set up sparkpost@renuo.ch as a main account for production

Reply:
I'd be more happy to keep a consistent naming. Yes, is not necessary to split testing and develop, but if it makes it easier to reason about it, then let's do it.

Reply:
Ok, so to be clear:

* I'd change the address on sparkpost+renuoapp@renuo.ch to sparkpost+develop@renuo.ch
* I'll create sparkpost+testing@renuo.ch
* I'll create sparkpost+master@renuo.ch, where we will migrate and merge the separate accounts for clients as subaccounts.

Reply:
Perfect!

Reply:
Good for me as well 👍